table/queries/merged_reduced_scans.sql

```diff
@@ -159,22 +170,24 @@ WITH AllScans AS (
     HAVING NOT STARTS_WITH(outcome, "setup/")
 )
 SELECT
-    Grouped.* except (country_code),
-    IFNULL(country_name, country_code) as country_name,
+    Grouped.* EXCEPT (country_code),
+    IFNULL(country_name, country_code) AS country_name,
     CASE
         WHEN (outcome = "complete/success") THEN 0
```

Comment:
I believe it's time to update how we output expected results.
It would be helpful to make it clearer what's expected.
In this case, we could have `expected/matches_control` and `expected/trusted_host:akamai`
The unexpected_count formula could use `STARTS_WITH('expected/')`

Reply:
Changed to 

`expected/match`
and
`expected/trusted_host:akamai`